src/applications/vaos/containers/AppointmentsPage.jsx

```diff
@@ -54,6 +65,36 @@ export class AppointmentsPage extends Component {
     }
   }
 
+  onSelectTab = tabIndex => {
+    const { selectedPastDateRange } = this.state;
+    this.setState({ tabIndex });
+
+    if (tabIndex === 1) {
```

Comment:
It might be worth making constants for the two tabs that point to the right index, to improve readability.